### DISCUSSION TITLE
SDC User Workshop online as well as in person

### DISCUSSION THREAD

**Discussion by violetacln** [sdcTools/UserSupport#229] [User Group]
The SDC User Workshop on the 20-th of September will be also online, not only in person, for the ones who cannot be in Paris at this date! Here are the details of the corresponding connection:
-	zoom link
https://inseefr.zoom.us/j/96673904462pwd=bFAxZ1dabjdnZmNaZVVkNUdVWkdYUT09
-	ID: 966 7390 4462 
-	Password : 483170
See you soon!

**Comment by auroradesantis** [comment]
Thanks Violeta 

Aurora De Santis

**Comment by Kyoshido** [comment]
Awesome, thank you.

**Comment by AncoHundepool** [comment]
I agree. I will be there.

 

Thanks,

Anco

 

Van: Jiří Novák 
Verzonden: 09 September 2022 12:08
CC: Subscribed ***@***.***>
Onderwerp: Re: [sdcTools/user-group] SDC User Workshop online as well as in
person (#24)

 

Awesome, thank you.

-
Reply to this email directly, view it on GitHub
<https://github.com/orgs/sdcTools/teams/user-group/discussions/24/comments/2
> , or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AFLUNXZSL6UXG43DACVO43DV5
MD73ANCNFSM6AAAAAAQF4H7EY> .
You are receiving this because you are subscribed to this thread.
<https://github.com/notifications/beacon/AFLUNX2RDMXVD6PGUIFPCKLV5MD73A5CNFS
M6AAAAAAQF4H7E2WGG33NNVSW45C7OR4XAZNTIRUXGY3VONZWS33OKBXXG5CSMVYGY6NKMNXW23L
FNZ2F62LEZYAAQSXI.gif> Message ID:
***@***.***
***@***.***> >

**Comment by spiliopv** [comment]
Good morning,

I tried to connect with the given link for zoom, but I see I am the only participant.
Is it still early or should I check something else?
Thank you in advance for your response.
Regards,

Vasiliki Spiliopoulou



From: Violeta Calian ***@***.***>
Sent: Tuesday, September 6, 2022 5:26 PM
Cc: Subscribed ***@***.***>
Subject: [Marketing Mail] [sdcTools/user-group] SDC User Workshop online as well as in person (#24)


The SDC User Workshop on the 20-th of September will be also online, not only in person, for the ones who cannot be in Paris at this date! Here are the details of the corresponding connection:

  *   zoom link
https://inseefr.zoom.us/j/96673904462pwd=bFAxZ1dabjdnZmNaZVVkNUdVWkdYUT09
  *   ID: 966 7390 4462
  *   Password : 483170
See you soon!

-

**Comment by AncoHundepool** [comment]
Good morning.

 

I am sorry but for me the link is not working.

I see the familiar zoom screen for one second and then 

The server could not be found????

 

Any idea?

 

Thanks,

Anco

 

 

Van: spiliopv 
Verzonden: 20 September 2022 08:45
CC: Anco Hundepool ***@***.***>; Comment
***@***.***>
Onderwerp: Re: [sdcTools/user-group] SDC User Workshop online as well as in
person (#24)

 

Good morning,

I tried to connect with the given link for zoom, but I see I am the only
participant.
Is it still early or should I check something else?
Thank you in advance for your response.
Regards,

Vasiliki Spiliopoulou



From: Violeta Calian ***@***.*** <mailto:***@***.***> >
Sent: Tuesday, September 6, 2022 5:26 PM
Cc: Subscribed ***@***.*** <mailto:***@***.***> >
Subject: [Marketing Mail] [sdcTools/user-group] SDC User Workshop online as
well as in person (#24)


The SDC User Workshop on the 20-th of September will be also online, not
only in person, for the ones who cannot be in Paris at this date! Here are
the details of the corresponding connection:

* zoom link
https://inseefr.zoom.us/j/96673904462pwd=bFAxZ1dabjdnZmNaZVVkNUdVWkdYUT09
* ID: 966 7390 4462
* Password : 483170
See you soon!

-
Reply to this email directly, view it on
GitHub<https://github.com/orgs/sdcTools/teams/user-group/discussions/24>, or
unsubscribe<https://github.com/notifications/unsubscribe-auth/ASMZPWZNHMP7AG
NMVA2GRMDV45H65ANCNFSM6AAAAAAQF4H7EY>.
You are receiving this because you are subscribed to this thread.Message ID:
***@***.******@***.*** <mailto:***@***.******@***.***> >>



-
Reply to this email directly, view it on GitHub
<https://github.com/orgs/sdcTools/teams/user-group/discussions/24/comments/4
> , or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AFLUNX2KG3TBWNYI4UBJ6ITV7
FMP5ANCNFSM6AAAAAAQF4H7EY> .
You are receiving this because you commented.
<https://github.com/notifications/beacon/AFLUNX7LE7NSNTQOP75KW5DV7FMP5A5CNFS
M6AAAAAAQF4H7E2WGG33NNVSW45C7OR4XAZNTIRUXGY3VONZWS33OKBXXG5CSMVYGY6NKMNXW23L
FNZ2F62LEZYAAQVKG.gif> Message ID:
***@***.***
***@***.***> >
